config/routes.rb

```diff
@@ -1,16 +1,28 @@
 Rails.application.routes.draw do
+  
+
   resources :routes
-  resources :trains
-  resources :railway_stations
+  resources :trains do
+    resources :cars, shallow: true
+    resources :econom_car, controller: 'cars', type: 'EconomCar', shallow: true
```

Comment:
Зачем для каждого типа вагонов создавать отдельный роут?

Reply:
Otherwise I'll get error:
<img width="1011" alt="screen shot 2017-10-30 at 8 43 15 am" src="https://user-images.githubusercontent.com/16690352/32171470-8eaa88e4-bd4e-11e7-8193-a2cba8a4687c.png">